backend/business-partner-agent/src/test/java/org/hyperledger/bpa/impl/aries/prooftemplate/ProofTemplateManagerTest.java

```diff
@@ -47,6 +50,8 @@
 
 @MicronautTest
 class ProofTemplateManagerTest {
+    @Inject
```

Comment:
why? it is never used anywhere

Reply:
used it before to test if the expected query is generated. but is dead code now. thanks for reminding.